pyproject.toml

```diff
@@ -35,6 +35,7 @@ fastapi = "^0.104.1"
 celery = {version = "^5.3.6", extras = ["redis"]}
 uvicorn = "*"
 pydantic-settings = "^2.2.1"
+label-studio-sdk = "^0.0.32"
```

Comment:
from comment above:
> # these are for the server
> # they would be installed as `extras` if poetry supported version strings for extras, but it doesn't
> # https://github.com/python-poetry/poetry/issues/834
> # they also can't be installed as a `group`, because those are for dev dependencies only and could not be included if this package was pip-installed

Are we ok with everyone (including external users/contributors) needing to clone this repo in order to use the server in stead of pip installing adala? I could see it going either way.

Reply:
Oh, I see what you mean - can do something like put the `label_studio_sdk` import inside this class, catch and surface the error from it not being available, and update our deployment setup to manually add the dep.

Reply:
done, not sure what needs to be done to install adala like this in deployment env:
`poetry install --with label-studio`

Reply:
Need to tell plate about it probably as well update our own dockerfile

Reply:
added to dockerfile

Reply:
@farioas we're adding the label studio SDK as an optional dep to Adala, is there anything that needs to be done manually here for our dev instance of adala to keep working? The dockerfile is already updated in this branch.